plugins/wdc/wdc-nvme.c

```diff
@@ -10576,7 +10576,7 @@ static int wdc_vs_temperature_stats(int argc, char **argv,
 	if (ret != 0)
 		goto out;
 
-   	/* convert from Kelvin to degrees Celsius */
+   	/* convert from kelvin to degrees Celsius */
```

Comment:
What about celsius? Shouldn't this also be lower cased?

Reply:
This is an exception.

> … the correct spelling of the name of the unit with the symbol °C is “degree Celsius” (the unit degree begins with a lower-case d and the modifier Celsius begins with an upper-case C because it is a proper name).
>
> —Section 5.3, _The International System of Units_, 9th Edition (2019)

Reply:
By the way NVMe specification revision 1.4c described as "in degrees Kelvin" for example in MNTMT. But it was changed as "in Kelvins" by revision 2.0c. Is this related to this issue? (Seems the specification also should be fixed to be lower case.)

Reply:
Thanks for the confirmation. Yes I can agree with the change to use 'K' instead of 'Kelvin'. Sorry a space is not used before '°C' so seems to need to fix it also.